packaging/macOS/pyinstaller.spec

```diff
@@ -2,71 +2,85 @@
 
 import os
 
- # Awesome hack to load `__version__`
+WORKSPACE_DIR = os.getenv("WORKSPACE_DIR")
+if WORKSPACE_DIR is None:
+    WORKSPACE_DIR = '../..'
+
+# Awesome hack to load `__version__`
 __version__ = None
-exec(open("../../parsec/_version.py", encoding="utf-8").read())
+exec(open(f"{WORKSPACE_DIR}/parsec/_version.py", encoding="utf-8").read())
+
+a = Analysis(
```

Comment:
The whole file has changed, but it seems really similar...
Can you explain what has changed ? (appart from tab len ^^)

Reply:
:+1: